platform/docs/docs/configuration/configurationFiles.md

```diff
@@ -119,6 +119,15 @@ Here are a list of some options available:
 - `omitQuotationForMultipartRequest`: Some servers (e.g., .NET) require the `multipart/related` request to be sent without quotation marks. Defaults to `false`. If your server doesn't require this, then setting this flag to `true` might improve performance (by removing the need for preflight requests). Also note that
 if auth headers are used, a preflight request is required.
 - `maxNumRequests`: The maximum number of requests to allow in parallel. It is an object with keys of `interaction`, `thumbnail`, and `prefetch`. You can specify a specific number for each type.
+- `modesConfiguration`: Allows overriding modes configuration.
+  - Example config:
+  ```js
+    modesConfiguration: {
+      ['@ohif/mode-longitudinal']: {
```

Comment:
why it is an array here?

Reply:
The array is just to define a property with special characters like the mode id (just a map).
I'll add more examples 👍

Reply:
I don't understand why you can't say [{ '@ohif/mode-longitudinal': {...}, { '@ohif/mode-segmentation':{...}]

and you need to say ['@ohif/mode-longitudinal']

Reply:
> I don't understand why you can't say [{ '@ohif/mode-longitudinal': {...}, { '@ohif/mode-segmentation':{...}]
> 
> and you need to say ['@ohif/mode-longitudinal']

I can use array but I think its a lot easier to use a map instead? so later I can just const config = modesConfiguration[modeId] instead of iterating the array looking for the config.

Reply:
Furthermore, I am trying to test this PR and when I go to save the configuration js file, the `['@ohif/mode-longitudinal']:` automatically changes to `'@ohif/mode-longitudinal':`.

Reply:
It's not required to use the brackets, this is just an explicit way of doing it. I'll update the docs.

Reply:
Is it worth mentioning in the docs that although the `modeConfiguration` is passed to the mode factory it is up to the particular mode itself if and how it applies the configuration? That is, for example, `basic-dev-mode` and `basic-test-mode` do not currently use this `modeConfiguration`. Just a thought.

Reply:
Agree, updated.